apps/cowswap-frontend/src/modules/tradeQuote/hooks/__snapshots__/useTradeQuotePolling.test.tsx.snap

```diff
@@ -1,33 +1,33 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`useTradeQuotePolling() When wallet is NOT connected Then  the "useAddress" field in the quote request should be 0x000...0000 1`] = `
+exports[`useTradeQuotePolling() When wallet is NOT connected Then the "useAddress" field in the quote request should be 0x000...0000 1`] = `
 {
-  "amount": "10000000",
+  "appData": "0xbace6e636671f2df6e59ef1eb4e63dce6b646a70c51b004ae5a07a9fbe4356fc",
+  "appDataHash": undefined,
   "buyToken": "0x91056D4A53E1faa1A84306D4deAEc71085394bC8",
-  "chainId": 1,
-  "fromDecimals": 18,
-  "isEthFlow": false,
+  "from": "0x0000000000000000000000000000000000000000",
   "kind": "sell",
+  "partiallyFillable": false,
   "priceQuality": "optimal",
-  "receiver": undefined,
+  "receiver": "0x0000000000000000000000000000000000000000",
+  "sellAmountBeforeFee": "10000000",
   "sellToken": "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
-  "toDecimals": 18,
-  "userAddress": undefined,
+  "validTo": undefined,
 }
 `;
 
 exports[`useTradeQuotePolling() When wallet is connected Then should put account address into "useAddress" field in the quote request 1`] = `
 {
-  "amount": "10000000",
+  "appData": "0xbace6e636671f2df6e59ef1eb4e63dce6b646a70c51b004ae5a07a9fbe4356fc",
+  "appDataHash": undefined,
   "buyToken": "0x91056D4A53E1faa1A84306D4deAEc71085394bC8",
-  "chainId": 1,
-  "fromDecimals": 18,
-  "isEthFlow": false,
+  "from": "0x333333f332a06ecb5d20d35da44ba07986d6e203",
   "kind": "sell",
+  "partiallyFillable": false,
   "priceQuality": "optimal",
   "receiver": "0x333333f332a06ecb5d20d35da44ba07986d6e203",
+  "sellAmountBeforeFee": "10000000",
   "sellToken": "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
-  "toDecimals": 18,
-  "userAddress": "0x333333f332a06ecb5d20d35da44ba07986d6e203",
+  "validTo": undefined,
```

Comment:
i don't love snapshots! i can't tell why we need to change things. I will have to believe these are good values

Reply:
🤷